packages/contracts/src/yield/strategy/TripleRateYieldStrategy.sol

```diff
@@ -3,14 +3,16 @@ pragma solidity ^0.8.20;
 
 import { ITripleRateContext } from "@credbull/yield/context/ITripleRateContext.sol";
 import { CalcSimpleInterest } from "@credbull/yield/CalcSimpleInterest.sol";
+// solhint-disable-next-line no-unused-import
 import { IYieldStrategy } from "@credbull/yield/strategy/IYieldStrategy.sol";
```

Comment:
I think we can remove these 2 lines if we are not using them

Reply:
Well spotted!
This is a problem with `@inheritdoc` . This contract extends `AbstractYieldStrategy`, but if I use that with `@inheritdoc` I get an error saying it had no functions to inherit the docs from. So I need to use `IYieldStrategy` for the `@inheritdoc`, and that requires a non-used import, which then requires the solhint rule disable.
Do you have a better idea?